src/MetaMorpho.sol

```diff
@@ -302,6 +302,7 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     /// @notice Removing a market requires the vault to have 0 supply on it; but anyone can supply on behalf of the
     /// vault so the call to `sortWithdrawQueue` can be griefed by a frontrun. To circumvent this, the allocator can
     /// simply bundle a reallocation that withdraws max from this market with a call to `sortWithdrawQueue`.
+    /// @dev Should not revert.
```

Comment:
After thinking about it, this comment alone does not make sense because we do want this function to revert for specific reasons

Reply:
Right, I'll remove it